testing/run-scripts/run_cloud.sh

```diff
@@ -64,7 +65,17 @@ then
     fi
     BANNER=`docker exec uproxy-sshd cat /banner`
   else
-    BANNER=`hostname`
+    # DigitalOcean's metadata API can tell us the region in which
+    # the machine is located:
+    #   https://developers.digitalocean.com/documentation/metadata/#metadata-api-endpoints
+    BANNER=`curl -s -m 2 http://169.254.169.254/metadata/v1/region`
+    if [ ! -z "$BANNER" ]
+    then
+      BANNER=`echo "$BANNER"|sed 's/ams./Netherlands/;s/sgp./Singapore/;s/fra./Germany/;s/tor./Canada/;s/nyc./New York, USA/;s/sfo./San Francisco, USA/;s/lon./UK/'`
```

Comment:
Nit: I notice this replaces the city with the country, except for US, where city is shown too. This would make sense as long as the US is the only country where DO has data centers in 2 different cities, but might they open up a data center in a different city of one of the existing countries? Either way, the "create droplet" UI uses city names for the labels the user selects from:
![screen shot 2016-01-28 at 10 57 15](https://cloud.githubusercontent.com/assets/64992/12649770/fb00e0f0-c5ad-11e5-9f59-480911d1d3a7.png)
Would it be more consistent, and no less clear, to do `sed 's/ams./Amsterdam/;s/sgp./Singapore/;s/fra./Frankfurt/;s/tor./Toronto/;s/nyc./New York/;s/sfo./San Francisco/;s/lon./London/'`?